pages/content-test/index.jsx

```diff
@@ -0,0 +1,13 @@
+import MarkdownPage from '../../packages/markdown-page'
+import Button from '../../packages/button'
+import generateGetStaticProps from '../../packages/markdown-page/server'
+
+function ContentTestPage(staticProps) {
+  return <MarkdownPage {...staticProps} components={{ Button }} />
+}
+
+export const getStaticProps = generateGetStaticProps({
+  pagePath: 'packages/content/fixtures/example-content.mdx',
+})
+
+export default ContentTestPage
```

Comment:
Made this page as a test to ensure `content` still had the intended appearance. Also helpful as a [preview for reviewing this PR](https://react-components-git-zsglobal-styles-cleanup-hashicorp.vercel.app/content-test). Plan is to remove it once things are ready to ship.